Tests/test_SeqIO_index.py

```diff
@@ -357,10 +353,28 @@ def tearDown(self):
         if os.path.isfile(self.index_tmp):
             os.remove(self.index_tmp)
 
+    @classmethod
+    def get_mode(cls, fmt):
```

Comment:
This needs a docstring, something like ``"""Determine file mode ("rt" or "rb") based on format."""``